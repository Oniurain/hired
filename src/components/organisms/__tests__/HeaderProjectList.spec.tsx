import * as React from 'react'
import { shallow } from 'enzyme'

import { HeaderProjectList } from '../'

describe('HeaderProjectList', () => {
  it('should render', () => {
    const subject = shallow(
      <HeaderProjectList />
    )

    expect(subject).toMatchSnapshot()
  })
})
