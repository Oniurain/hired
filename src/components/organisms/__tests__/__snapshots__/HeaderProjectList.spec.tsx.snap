// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderProjectList should render 1`] = `
<styled.header>
  <Styled(WithRoute[BaseLink])
    routeName="PROJECT_NEW"
  >
    <Styled(Icon) />
  </Styled(WithRoute[BaseLink])>
  <styled.h1>
    Projects
  </styled.h1>
</styled.header>
`;
